docs/usage/storage.rst

```diff
@@ -0,0 +1,48 @@
+Storage
+=======
+
+.. todo::
+   Intro sentence.
+
+Volumes
+-------
+
+Any workload needs different volumes with different storage capacities
+to store its components. Following the total storage capacity needed,
+establish a number of volumes and their capacities for each node, as
+exemplified below:
+
+.. code-block:: yaml
+
+    metalk8s_lvm_lvs_vg_metalk8s: ['/dev/vdb']
```

Comment:
This YAML seems ill-formatted.

Reply:
Why? I copy pasted the existing one a few versions ago